metals/src/main/scala/scala/meta/internal/metals/MultilineStringFormattingProvider.scala

```diff
@@ -3,14 +3,14 @@ package scala.meta.internal.metals
 import org.eclipse.lsp4j.DocumentOnTypeFormattingParams
 import org.eclipse.lsp4j.Range
 import org.eclipse.lsp4j.TextEdit
+
 import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
 import scala.meta.inputs.Input
 import scala.meta.internal.metals.MetalsEnrichments._
 import scala.meta.tokens.Token
-import scala.meta.tokens.Token.Constant
+import scala.meta.tokens.Token.{Constant, Ident, Interpolation}
```

Comment:
Could you split the imports into separate lines? That's the convention we have in Metals. Also, scalafix is failing due to the fact that one of those is no longer used.

Reply:
yes. thanks